src/tasks.rs

```diff
@@ -66,6 +66,7 @@ pub struct DocumentAdditionOrUpdate {
 pub struct DocumentDeletion {
     pub provided_ids: Option<usize>,
     pub deleted_documents: Option<usize>,
+    pub original_filter: String,
```

Comment:
The normal document deletion shouldn’t work anymore with your code.
This is the type meilisearch can return:

```suggestion
    pub provided_ids: usize,
    pub deleted_documents: Option<usize>,
    pub original_filter: Option<String>,
```

Reply:
`provided_ids` is not returned when calling the [delete all documents route](https://www.meilisearch.com/docs/reference/api/documents#delete-all-documents).

I'll put both in optional